packages/webapp/pages/posts/[id]/index.tsx

```diff
@@ -132,6 +132,64 @@ const PostPage = ({ id, initialData }: Props): ReactElement => {
   };
 
   const seoComponent = <NextSeo {...seo} />;
+  const schemaComponent = (
+    <script
+      type="application/ld+json"
+      dangerouslySetInnerHTML={{
+        __html: JSON.stringify({
```

Comment:
Could we extract this as a function and store it in a separate file, that way we don't "overload" this file with lots of code?

Reply:
Just wanted to write, I would even pull it in a component 👍

Reply:
Yup, that was my idea, but I'm running out of working time today 😅 

Hence why I commented: https://github.com/dailydotdev/apps/pull/2706#discussion_r1507598648

Reply:
Or use `next-seo` https://github.com/dailydotdev/apps/pull/2706#discussion_r1507612410